multiversx_sdk/wallet/user_test.py

```diff
@@ -119,8 +119,8 @@ def test_sign_transaction():
     tx = Transaction(
         nonce=89,
         value=0,
-        receiver="erd1spyavw0956vq68xj8y4tenjpq2wd5a9p2c6j8gsz7ztyrnpxrruqzu66jx",
-        sender="erd1qyu5wthldzr8wx5c9ucg8kjagg0jfs53s8nr3zpz3hypefsdd8ssycr6th",
+        receiver=Address.new_from_bech32("erd1spyavw0956vq68xj8y4tenjpq2wd5a9p2c6j8gsz7ztyrnpxrruqzu66jx"),
```

Comment:
Make sure to update the specs on this matter, as well.

How will we handle this in JS?

Reply:
If you refer to the `Transaction` specs, they are updated [here](https://github.com/multiversx/mx-sdk-specs/blob/main/core/transaction.md). Regarding handling this in the js sdk, I am not sure what you mean.